package.json

```diff
@@ -1,32 +0,0 @@
-{
-  "name": "fac-mentorship-schedule",
-  "version": "1.0.0",
-  "description": "A web app to schedule mentors for every week in the cohorts",
-  "main": "src/server.js",
-  "dependencies": {
-    "env2": "^2.2.0",
-    "istanbul": "^0.4.5",
-    "nodemon": "^1.11.0",
-    "pg": "^7.1.2",
-    "shot": "^3.4.2",
-    "tap-spec": "^4.1.1",
-    "tape": "^4.8.0"
-  },
```

Comment:
istanbul, nodemon, shot, tap-spec and tape should be dev dependencies as they are not necessary to run the app, but to run tests